app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +32,13 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function authenticated(Request $request, $user)
+    {
+        Login::create([
+            'user_id' => $user->id,
```

Comment:
Can IP be added to this as well. I believe you can grab it via `$request->ip();`

Reply:
Sure will add it.